dot/parachain/statement-distribution/statement_distribution_test.go

```diff
@@ -0,0 +1,22 @@
+package statementdistribution
+
+// import (
+// 	parachainutil "github.com/ChainSafe/gossamer/dot/parachain/util"
+// )
+
+// func createStatementDistribution() (*StatementDistribution, chan parachainutil.NetworkBridgeTxMessage) {
```

Comment:
This is ok to have, just the return that should be a `chan any`
```suggestion
// func createStatementDistribution() (*StatementDistribution, chan any) {
```